_rules/aria-state-or-property-not-prohibited.md

```diff
@@ -0,0 +1,150 @@
+---
+id: 
+name: ARIA state or property is not prohibited
+rule_type: atomic
+description: |
+  This rule checks that WAI-ARIA states or properties are not prohibited for the element they are specified on. 
+accessibility_requirements:
+  wcag-technique:ARIA5: # Using WAI-ARIA state and property attributes to expose the state of a user interface component
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  aria12:state_property_processing:
+    title: ARIA 1.2, 8.6 State and Property Attribute Processing
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: satisfied
+  wcag20:1.3.1: # Info and Relationships (A)
+    secondary: true
+  wcag20:4.1.2: # Name, Role, Value (A)
+    secondary: true
+input_aspects:
+  - Accessibility Tree
+  - CSS styling
+  - DOM Tree
+acknowledgments:
+  authors:
+    - Anne Thyme Nørregaard
+    - Jean-Yves Moyen
+    - Trevor Bostic
```

Comment:
Should I keep all of the authors? Much of the content was created by Anne and Jean-Yves so I want to make sure proper credit is given.